CODE-OF-CONDUCT.md

```diff
@@ -34,6 +34,79 @@ In keeping with this commitment, we offer the following guidelines:
      in [Security Reviews:
      Outcome][review-outcome]
 
+
+# Incident handling and escalation
+
+Content for the purposes of the code of conduct as well as incident is defined
+not only as published or draft content but also online discourse, such as slack
+messages or emails, and interactions at in-person events.  If an incident
+involving community conduct occurs, please follow the guidelines below on how to
+handle and report the issue:
+
+- If you see content that clearly does not meet the official Code of Conduct,
+  please send an e-mail to the Co-Chair/TL mailing list
```

Comment:
Consider including some mechanism in case the incident is regarding the behavior of a co-chair?